### DISCUSSION TITLE
How is the configOverridesFiles in the Helm chart supposed to work?

### DISCUSSION THREAD

**Discussion by paulinjo** [apache/superset#19601] [Q&A / Help]
Where do the python files need to be in order to have them appended to the superset_config?

I've tried mounting them in the container on the pythonpath but without any luck.

**Comment by caponord** [comment]
Anyone that has got this working?  Tried various things I only get a comment in the superset_config.py file instead of the contents of the file. 

![image](https://user-images.githubusercontent.com/38348999/189724428-fdfa97b5-e992-4a28-8b56-2e97fc6c46d9.png)

**Comment by caponord** [comment]
The issue for me was this https://github.com/helm/helm/issues/3276

So my config now looks like:

![image](https://user-images.githubusercontent.com/38348999/189734243-b78d1c11-bff1-4e4b-a3ba-e3bca0e72def.png)

Earlier I had a relative path to the file in the value for extend_custom_roles and hence it didn't work. It works if the file is in the same directory as the chart.

**Reply by shauryagoel** [reply]
Is this still working for you? I am doing the same thing as you mentioned in this comment, but, I am seeing a comment just like you said in the previous comment. Is this command supposed to mount the file mentioned in the value to `/app/pythonpath`?

**Comment by ameen7626** [comment]
I am also looking for a solution
```
configOverridesFiles:
       redirection: redirection_config.py
```       
This doesn't work for me even though the python file is in same directory as my value file

**Reply by PaleNeutron** [reply]
first download chart, then put it at the root of the decompressed directory which contains 

```
Chart.lock  charts  Chart.yaml  README.md  README.md.gotmpl   templates  values.yaml
```
